src/Api/Workspaces/Projects.php

```diff
@@ -71,7 +71,7 @@ public function show(string $project, array $params = [])
      *
      * @return array
      */
-    public function update(string $project, array $params = [])
```

Comment:
Please don't change this. We have params like this out of convention. If we want to review that, we'd want to do it for all cases, not just one, and not in this PR.